### DISCUSSION TITLE
Add translation to About

### DISCUSSION THREAD

**Discussion by mehr32** [searxng/searxng#2687] [Q&A]
Hello ! me Again :) , I have a question:
1- How should I add a new language to the section about searxng?
1.1- How should I add the codes for testing? (I added it but it didn't work, I realized that it only accepts supported languages. That's why I want to add another language on my computer as a test)

**Comment by dalf** [comment]
In the git repository, look at the [`searx/infopage` directory](https://github.com/searxng/searxng/tree/master/searx/infopage), there is one subdirectory per language. For example, the about page in English is here: [searx/infopage/en/about.md](https://github.com/searxng/searxng/blob/master/searx/infopage/en/about.md).

So to add a new language: create a directory with the language code you want to add (ISO 639-1 Code), copy paste the *.md files from the `en` directory, and you can start to translate the content (the files are written in markdown, with some special sequence in double bracket `{{ .... }}` : just ignore them).

SearXNG is going to detect the new files when it starts.

Once done, you can submit a pull request.

**Reply by mehr32** [reply]
I did this, but it is not applied, that is, when the searxng language is loaded on that tab, English is still loaded. Even if I change the url manually /:
